install.sh

```
@@ -136,46 +136,57 @@ install_lookup_tables() {
+    "$HOME/.ezkl/bin/ezkl"
+    "$HOME/.config/.ezkl/ezkl"
+    "$HOME/.config/.ezkl/bin/ezkl"
+    "$HOME/.config/.ezkl"
+    "$HOME/.local/bin/ezkl"
+    "$HOME/.cargo/bin/ezkl"
+  )
+
+  for p in "${potential_paths[@]}"; do
+    if [[ -x "$p" ]]; then
+      info "Found EZKL at $p. Adding to PATH."
+      export PATH="$(dirname "$p"):$PATH"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)